apps/meteor/client/hooks/useIsEnterprise.ts

```diff
@@ -1,8 +1,13 @@
 import { useEndpoint } from '@rocket.chat/ui-contexts';
 import { useQuery } from '@tanstack/react-query';
 
-export const useIsEnterprise = (): boolean => {
+export const useIsEnterprise = (): { isEnterprise: boolean; isLoading: boolean } => {
```

Comment:
Please, don't hide the return of `useQuery`. Always return all the properties. There's a helper type from `@tanstack/react-query` called `UseQueryResult`, that you can use to wrap the type that comes in `data`. See `apps/meteor/client/hooks/useRoomInfoEndpoint.ts` for more context.

Reply:
Done!